bin/zowe-configure-component.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/sh
+DEFAULT_TARGET_DIR=/opt/zowe/extensions
+
+if [[ -z ${ZOWE_ROOT_DIR} ]]
+then
+  export ZOWE_ROOT_DIR=$(cd $(dirname $0)/../;pwd)
+fi
+
+# Try and source the file utils if it exists
+if [[ -f "${ZOWE_ROOT_DIR}/bin/utils/file-utils.sh" ]]
+then
+    . ${ZOWE_ROOT_DIR}/bin/utils/file-utils.sh
+fi
+
+error_handler(){
+    print_error_message "$1"
+    exit 1
+}
+
+install_component(){
+
+    # Represents the current value of EXTERNAL_COMPONENT variable in instance.env
+    EXT_COMP_VAL=$(eval "grep '^EXTERNAL_COMPONENTS=' $INSTANCE_DIR/instance.env | cut -f2 -d= | cut -f1 -d#")
+
+    if [ "$EXT_COMP_VAL" = " " ]; then
```

Comment:
Instead of adding full component lifescycle script path to `EXTERNAL_COMPONENTS `, we can add component name as value, and also enforce `ZWE_EXTENSION_DIR` to be target_dir parameter value.